### DISCUSSION TITLE
do not restrict home feed to people starring my repos unconditionally

### DISCUSSION THREAD

**Discussion by giulianopz** [community/community#118546] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Very often I land on my home feed and I can only see people who starred my repos (or something similar related to me) without the possibility to switch back to the 'normal' view of the feed, i.e. what people are doing out there that is not related to me, which is the only stuff I care. I struggle to find a way to control this feed behavior, but it seems not possible to me. That's annoying.
Please, add the possibility for GitHub users to switch back and forth among these different views of the home feed (e.g. via a button).

**Comment by Enigmatisms** [comment]
Indeed, and what's even more annoying is that developers seem to be indifferent to these "feedbacks".

**Reply by giulianopz** [reply]
Ironically, after opening this discussion, the problem is occurring more and more frequently. GitHub seems well advanced in the enshittification process.
